src/server/pfs/server/driver.go

```diff
@@ -471,11 +471,19 @@ func (d *driver) listProject(ctx context.Context, cb func(*pfs.ProjectInfo) erro
 }
 
 // TODO: delete all repos and pipelines within project
-func (d *driver) deleteProject(txnCtx *txncontext.TransactionContext, project *pfs.Project, force bool) error {
-	if err := d.projects.ReadWrite(txnCtx.SqlTx).Delete(pfsdb.ProjectKey(project)); err != nil {
-		return errors.Wrapf(err, "delete project %q", project.Name)
+func (d *driver) deleteProject(ctx context.Context, req *pfs.DeleteProjectRequest) error {
```

Comment:
Not a huge fan of passing in the `DeleteProjectRequest` here rather than destructuring to the important bits. On the one hand there is a benefit to using a struct as an argument to act kinda like keyword args, but on the other a function call really ought to read naturally-ish.